app/views/RoomView/styles.js

```diff
@@ -46,5 +46,16 @@ export default StyleSheet.create({
 	previewMode: {
 		fontSize: 16,
 		...sharedStyles.textMedium
+	},
+	scrollButton: {
+		position: 'absolute',
+		bottom: 70,
```

Comment:
This position should be based on device.
Take a look how it is hidden behind the keyboard on iPhone 11 Pro:
<img src="https://user-images.githubusercontent.com/804994/74954144-6e35fb80-53e1-11ea-9acc-e3147d3ad2a0.png" width="350" />